spark-doris-connector/spark-doris-connector-base/src/main/java/org/apache/doris/spark/client/write/AbstractStreamLoadProcessor.java

```diff
@@ -85,19 +83,19 @@ public abstract class AbstractStreamLoadProcessor<R> extends DorisWriter<R> impl
 
     private final Map<String, String> properties;
 
-    private final String format;
+    private final DataFormat format;
 
     protected String columnSeparator;
 
-    private String lineDelimiter;
+    private byte[] lineDelimiter;
 
     private final boolean isGzipCompressionEnabled;
 
     private String groupCommit;
 
     private final boolean isPassThrough;
 
-    private PipedOutputStream output;
+    private StreamLoadEntity output;
```

Comment:
Why do we need to change this?

Reply:
For better performance

Before:

Reply:
Is this performance test based on this PR? https://github.com/apache/doris-spark-connector/pull/294

Reply:
I saw the interval mistake. This PR was originally intended to fix this issue as well, but it had already been fixed by the time I submitted the PR. However, even after fixing this interval mistake, the performance is still much worse compared to version 1.3.2. The test results in our actual scenario are as follows:
1. version with interval mistake: 100.8 GiB, Time > 24H
2. version with interval mistake fixed: 100.8 GiB, Time: 23 min
3. version with this PR, Time: 12 min
4. version 1.3.2, Time: 10 min

Reply:
The key modifications: apart from StreamLoadEntity, also include the class RowConvertors.scala

Reply:
Thank you for your test results, PTAL  @gnehil